packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/overview/index.md

```diff
@@ -8,6 +8,10 @@ This guide walks you through creating a Groups claim for an OpenID Connect clien
 
 Additionally, you can create a [dynamic](/docs/guides/customize-tokens-dynamic/) or [static](/docs/guides/customize-tokens-static) allow list when you need to set group allow lists on a per-application basis using both the Org Authorization Server and a Custom Authorization Server.
 
+The examples in this guide use the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) for quick testing. The example requests to the <GuideLink link="../add-groups-claim-org-as">Org Authorization Server</GuideLink> include only `id_token` as the `response_type` value. This means that the requests are for a [fat ID token](/docs/concepts/api-access-management/#tokens-and-scopes), and the ID token is the only token included in the response. The ID token contains any groups assigned to the user that signs in when you include the `groups` scope in the request.
```

Comment:
Should we move this to the `guides/customize-tokens-groups-claim/add-groups-claim-custom-as`? And in there you can mention how to change the configuration to always include group claim